src/node/nodestate.h

```diff
@@ -1651,6 +1651,7 @@ namespace ccf
         *network.tables->get<pbft::RequestsMap>(pbft::Tables::PBFT_REQUESTS),
         *network.tables->get<pbft::PrePreparesMap>(
           pbft::Tables::PBFT_PRE_PREPARES),
+        network.signatures,
```

Comment:
Any reason why this map is passed in like this while the request and pre-prepare tables are passed in with `*network.tables->get<>()`?

Reply:
No just left over code, I update it